lesson-10/public/app.js

```diff
@@ -0,0 +1,58 @@
+$('#button').click(() => {
+  const memeName = $('#myName').val();
+  const memeComment = $('#myComment').val();
+  $.ajax({
+    type: 'POST',
+    url: '/sendmessage',
+    contentType: 'application/json',
+    data: JSON.stringify({
+      memeName,
```

Comment:
same here, if you are only going to use memeName and memeComment once, then try to avoid assigning it to a variable and just use it directly. Check your other possible misuse on this as well